docs/FeatureManagement/FeatureFlag.v2.0.0.schema.json

```diff
@@ -0,0 +1,345 @@
+              "description": "A list of objects, each containing a variant name and list of groups for which that variant should be used.",
+              "items": {
+                  "$id": "#/properties/allocation/properties/group/items",
+                  "type": "object",
+                  "title": "Group Allocation",
+                  "required": [
+                      "variant",
+                      "groups"
+                  ],
+                  "properties": {
+                      "variant": {
+                          "$id": "#/properties/allocation/properties/group/items/properties/variant",
+                          "type": "string",
+                          "title": "Group Allocation Variant",
+                          "description": "The name of the variant to use if the group allocation matches a group the current user is in.",
+                          "pattern": "^(.*)$"
+                      },
+                      "groups": {
+                          "$id": "#/properties/allocation/properties/group/items/properties/groups",
+                          "type": "array",
+                          "title": "Group Allocation Groups Collection",
+                          "description": "Collection of groups where if the current user is in any of these groups, the variant specified in the group allocation is used.",
+                          "items": {
+                              "type": "string"
+                          }
+                      }
+                  }
+              }
+          },
+          "percentile": {
+              "$id": "#/properties/allocation/properties/percentile",
+              "type": "array",
+              "title": "Percentile Allocation Collection",
+              "description": "A list of objects, each containing a variant name and percentage range for which that variant should be used.",
+              "items": {
+                  "$id": "#/properties/allocation/properties/percentile/items",
+                  "type": "object",
+                  "title": "Percentile Allocation",
+                  "required": [
+                      "variant",
+                      "from",
+                      "to"
+                  ],
+                  "properties": {
+                      "variant": {
+                          "$id": "#/properties/allocation/properties/percentile/items/properties/variant",
+                          "type": "string",
+                          "title": "Percentile Allocation Variant",
+                          "description": "The name of the variant to use if the calculated percentile for the current user falls in the provided range.",
+                          "pattern": "^(.*)$"
+                      },
+                      "from": {
+                          "$id": "#/properties/allocation/properties/percentile/items/properties/from",
+                          "type": "number",
```

Comment:
QQ: is there a way to specify the range?

Reply:
Not sure what you mean, like instead of using `from` and `to` as the range?

Reply:
The from and to must be in the range of 0 and 100. Is there a way to specify it?

Reply:
Oh right, I think jimmy commented mentioning I should do that with `minimum` and `maximum` for those values. I've added that in the last commit.